api/environments/identities/views.py

```diff
@@ -114,26 +115,17 @@ class SDKIdentitiesDeprecated(SDKAPIView):
     # identifier is in a path parameter
     def get(self, request, identifier, *args, **kwargs):  # type: ignore[no-untyped-def]
         # if we have identifier fetch, or create if does not exist
```

Comment:
I think this comment is stale?

Reply:
I guess so. Maybe _my_ comment should have been: "I don't think this comment is adding anything anymore (or really ever was)".

Reply:
Agree with _your_ comment. I would actually prefer to refactor the entirety of this, though I'm afraid this PR has become too much of a chonky boi thanks to coverage complaining about me messing with ancient technology. Are you OK with taking that [and a lot of other] effort to a new issue?

Reply:
> Are you OK with taking that [and a lot of other] effort to a new issue?

Sure am 👍

Reply:
Can you add a link to the issue here once created, and resolve this conversation?

Reply:
Here we go: https://github.com/Flagsmith/flagsmith/issues/6021
Feel free to add more detail.